driver-core/src/test/unit/com/mongodb/internal/connection/ServerDiscoveryAndMonitoringTest.java

```diff
@@ -51,6 +51,9 @@ public class ServerDiscoveryAndMonitoringTest extends AbstractServerDiscoveryAnd
     public ServerDiscoveryAndMonitoringTest(final String description, final BsonDocument definition) {
         super(definition);
         this.description = description;
+        // Unified and monitoring tests have their own test runners
```

Comment:
Is there a way to do this without skips?  I'm concerned that it will be confusing, as it overloads the meaning of skipping a test, which is typically done because the driver has not implemented some feature or is otherwise incompatible with the test.

Reply:
Filtered the data instead.